labelbox/schema/ontology.py

```diff
@@ -145,13 +147,26 @@ def __post_init__(self):
                 "removed in a future release. Dropdown will also "
                 "no longer be able to be created in the Editor on 3/31/2022.")
 
-    @property
-    def name(self) -> str:
-        return self.instructions
+        if self.name is None:
+            msg = (
+                "When creating the Classification feature, please use “name” "
+                "for the classification schema name, which will be used when "
+                "creating annotation payload for MAL and Labelimport. "
```

Comment:
`creating annotation payload for Model-Assisted Labeling Import and Label Import`. Sorry for being a stickler. Just want to make sure we clean up any wording that goes out to customers

Reply:
I have updated the message.